cli/src/vrf.ts

```diff
@@ -77,6 +77,13 @@ export function listHandler(print?: boolean) {
     if (!(await isOraklNetworkApiHealthy())) return
 
     try {
+      if (!chain) {
+        const result = (await axios.get(VRF_ENDPOINT)).data
```

Comment:
If `data` is not returned by VRF endpoint, it would crash when we try to access it.

```suggestion
        const result = (await axios.get(VRF_ENDPOINT))?.data
```